bitcoin-explained/security-issue-burying-bitcoin-soft-forks.md

```diff
@@ -165,93 +174,85 @@ So there's only one historical transaction that happened before Taproot activate
 And more importantly, somebody else took that money.
 And that taking of the money would not be compliant with the modern Taproot rules.
 
-Speaker 0: 00:05:46
+Aaron van Wirdum: 00:05:46
 
 Yes.
 Well, actually, I'll have to correct you there a little bit.
 So actually several people sent money to a Taproot address before Taproot was active.
 
-Speaker 1: 00:05:56
+Sjors Provoost: 00:05:56
 
 But only one took it.
 
-Speaker 0: 00:05:57
+Aaron van Wirdum: 00:05:57
 
 Yes.
 It's only been taken one time.
 Now Sending to such an address was always okay, of course, and is still okay, and that was never a problem.
 It's a taken from that address that since Taproot is active, you cannot do that anymore.
 But before Taproot was active, you could just take that money.
 
-Speaker 1: 00:06:15
+Sjors Provoost: 00:06:15
 
 That's right.
 Okay.
 And so this presents an opportunity to look at the Bitcoin Core source code, which currently just says, wait, when you see the blockchain for the first time, you think the whole thing, you look at the signaling, and then after a while, you've seen that the speedy trial activated, and now you're going to apply the new rules to the new blocks.
 What you can do instead is say, don't worry about the signaling, just apply the taproot rules from the Genesis block onwards, pretend that taproot has always existed with one exception.
 And that exception would be hard-coded in the source code, saying, well, Zaproot is always active except for this specific block.
 
-Speaker 0: 00:06:53
+Aaron van Wirdum: 00:06:53
 
 Yes, and that block would be the block that includes the transaction that I just mentioned that took these coins.
 
-Speaker 1: 00:07:00
+Sjors Provoost: 00:07:00
 
-And
+Yeah.
 
-Speaker 0: 00:07:00
+Aaron van Wirdum: 00:07:00
 
-they were sent to Brink, actually.
+They were sent to Brink, actually.
 There's a blog post that was, while we're discussing it, let me just mention that real quick.
 So it was actually taken by, or at least the Bitcoin developer named 0xb10c.
-He sort of initiated it and then it was included by F2Pool, and it was sent, so these coins that were sent to Brink at the time.
+He sort of initiated it and then it was included by `F2Pool`, and it was sent, so these coins that were sent to Brink at the time.
 
-Speaker 1: 00:07:27
+Sjors Provoost: 00:07:27
 
-Oh, that's very nice.
+That's very nice.
 
-Speaker 0: 00:07:27
+Aaron van Wirdum: 00:07:27
 
-Not very important in the context of the rest of this episode, but a
+Not very important in the context of the rest of this episode, but a little detail.
 
 ## Exception block
 
-Speaker 1: 00:07:31
+Sjors Provoost: 00:07:31
```

Comment:
Should I replace it with BIP16?

Reply:
I noticed that on line 678 you changed "BIP, Bitcoin Improvement Proposal." to just "BIP16"
Please reverse it because the former aligns with the source material